db/post_migrate/20200602034543_make_invite_user_id_redeemed_at_read_only.rb

```diff
@@ -0,0 +1,11 @@
+class MakeInviteUserIdRedeemedAtReadOnly < ActiveRecord::Migration[6.0]
```

Comment:
This should be done before the migration to copy `invites` over to `invited_users`. Otherwise, the running applications will still be writing to the `invites` table before the new code changes have been deployed.

Reply:
Okay, I moved the migration block here just after making the columns readonly.